docs/KNOWN_ISSUES.md

```diff
@@ -35,3 +35,4 @@ let api_gws = Resources.*[ Type == 'AWS::ApiGateway::RestApi' ]
 3. When performing `!=` comparison, if the values are incompatible like comparing a `string` to `int`, an error is thrown internally but currently suppressed and converted to `false` to satisfy the requirements of Rust’s [PartialEq](https://doc.rust-lang.org/std/cmp/trait.PartialEq.html). We are tracking to release a fix for this issue soon.
 4. `exists` and `empty` checks do not display the JSON pointer path inside the document in the error messages. Both these clauses often have retrieval errors which does not maintain this traversal information today. We are tracking to resolve this issue. 
 5. Currently, for `string` literals, Guard does not support embedded escaped strings. We are tracking to resolve this issue soon.
+6. We have support for built-in functions, however, this is currently limited to assignment of the return value to a variable. For example, we can use a function and assign its result to a variable such as `let no_of_instances = count(Instances.*)` and then this variable can be used elsewhere in the conditions such as `%no_of_instances < 2`. However, we **cannot** re-write the same condition as `count(Instances.*) < 2` at this point of time.
```

Comment:
I think we can explain this more thorough. We should probably mention that anytime a function is called it needs to be assigned before it is used anywhere. I.e parameterized rules, other function params, clauses, etc..

Reply:
added explanation